lib/urbanopt/geojson/schema/thermal_junction_properties.json

```diff
@@ -60,9 +60,7 @@
     }
   },
   "required": [
-    "type",
-    "junction_type",
```

Comment:
is this no longer needed?

Reply:
we aren't using it for GHE workflows

Reply:
For GHE , we are using id, type, buildingID, DSID, Is_ghe_start_loop

Reply:
@nllong  @amyeallen1 can you comment on this? I believe we added junction_type on thermal junctions with enum values of: DES, ETS, or Valve.  Is this unused and can/should we remove it (from the required list of fields) for the DES workflow?

Reply:
Thanks, Katherine and Tanushree--I think it would be worth keeping, and making non-required. I can't quite remember what case "DES" is supposed to represent, but maybe we could revisit the options in the future. @kflemin @tanushree04 @nllong

Reply:
Agreed. Keep until we know we don't need it. @amyeallen1  -- maybe we can revisit creating the ATTEN network in the geojson first to make sure it works for a real case?

Reply:
Yes, good idea! @nllong

Reply:
thanks all! we will keep it but make it not required